src/i18n.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+/* istanbul ignore file */
+
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import translation from './locales/en/translation.json';
```

Comment:
Don't copy the file to assisted-ui. The translations should always be imported from assisted-ui-lib. They are available from
```
import locales from 'openshift-assisted-ui-lib/locales'
```